enos/provider/g5k.py

```diff
@@ -54,10 +54,7 @@ def init(self, conf, force_deploy=False):
         return (roles, network, (network_interface, external_interface))
 
     def destroy(self, env):
-        conf = load_config(env['config'],
-                           self.topology_to_resources,
-                           self.default_config())
-        provider_conf = conf['provider']
+        provider_conf = env['config']['provider']
```

Comment:
This works, now, thanks to the `finally` block in the enostask that saves the env during the up phase, even if the up phase fails. But, this will still fail if you do an `enos destroy` before an `enos up`. Could you please add a test that checks, at least, that the `'provision'` key exists and displays a log.error otherwise?